### DISCUSSION TITLE
How to categorize algorithms?

### DISCUSSION THREAD

**Discussion by ghost** [MakeContributions/DSA#64] [Ideas]
It's not quite clear that how should we categorize different algorithms.

For example I've some algorithms for counting inversions in an array and finding closest pair of 2d points but I'm not sure in which category should I put them.  They are both divide and conquer algorithm but if I create a new category for d&c algorithm them then should we move karatsuba, merge sort and quick sort their because they are also d&c algorithms.

We should also update the README with these kind of instruction.

Issue regarding this => #65

**Comment by ming-tsai** [comment]
Hi @atin,
In your case could be on the searching category

**Comment by Arsenic-ATG** [comment]
It should be up to contributor where the algorithm is perfectly fitting, in case there exists a better fitting place for that algorithm, the contributor can be told about the same via review or comments on their PR.
